### PR TITLE
Translated some Texts to Spanish

### PR DESCRIPTION
**Pull request by TheKillerTacos**
Translated the UnTraslated :P from some texts,

And also can you change the Config.yml if merge happens:

`# Spanish message resource by @iksaku and @JoahDave (Github) `
To
`# Spanish message resource by @iksaku, @JoahDave and @TheKillerTacos (Github) `
Or something

### CODE REVIEW

File: resources/messages-sp.yml [hunk 1]
```diff
@@ -138,9 +139,9 @@ cmds:
       usrinfo_username: "NOMBRE DE USUARIO: %var0%"
       usrinfo_status: "ESTADO: %var0%"
       usrinfo_ip: "IP: %var0%"
-      usrinfo_uuid: "UNIQUE ID: %var0%"
+      usrinfo_uuid: "ID UNICA: %var0%"
       usrinfo_group: "GRUPO: %var0%"
-      unknown: "UNKNOWN"
+      unknown: "DESCONOSIDO"
```
Comment (iksaku) [inline resources/messages-sp.yml]: Desconocido*

File: resources/messages-sp.yml [hunk 2]
```diff
@@ -138,9 +139,9 @@ cmds:
       usrinfo_username: "NOMBRE DE USUARIO: %var0%"
       usrinfo_status: "ESTADO: %var0%"
       usrinfo_ip: "IP: %var0%"
-      usrinfo_uuid: "UNIQUE ID: %var0%"
+      usrinfo_uuid: "ID UNICA: %var0%"
       usrinfo_group: "GRUPO: %var0%"
```
Comment (iksaku) [inline resources/messages-sp.yml]: Única*

**Comment by TheKillerTacos** [comment]
haha Sorry, using my mini laptop, and i was on a hurry, :P

**Comment by 64FF00** [comment]
@TheKillerTacos @iksaku https://github.com/PurePlugins/PurePerms/commit/6349bea45e1c6b69a18c1b8af9080f2a4d363c08

**Pull request closed without merging**